advanced/Scripts/piholeDebug.sh

```diff
@@ -1076,18 +1076,27 @@ show_db_entries() {
     IFS="$OLD_IFS"
 }
 
+show_groups() {
+    show_db_entries "Groups" "SELECT * FROM \"group\"" "4 100 7 10 13 50"
+}
+
 show_adlists() {
     show_db_entries "Adlists" "SELECT * FROM adlist" "4 100 7 10 13 50"
+    show_db_entries "Adlist groups" "SELECT * FROM adlist_by_group" "4 100 7 10 13 50"
 }
 
 show_whitelist() {
     show_db_entries "Exact whitelist" "SELECT * FROM whitelist" "4 100 7 10 13 50"
     show_db_entries "Regex whitelist" "SELECT * FROM regex_whitelist" "4 100 7 10 13 50"
+    show_db_entries "Exact whitelist groups" "SELECT * FROM regex_whitelist_by_group" "4 100 7 10 13 50"
+    show_db_entries "Regex whitelist groups" "SELECT * FROM whitelist_by_group" "4 100 7 10 13 50"
```

Comment:
What do you think of putting the group directly after the list table it references? Or is separating the lists from the group tables better?

Reply:
Yes, we can move them like you suggested.